app/views/games/_index.html.erb

```diff
@@ -0,0 +1,55 @@
+<% if @games.present? %>
+
+  <div class="row">
+    <% @games.each do |g|  %>
+    <div class="col-md-4">
+
+        <div class="panel-result col-md-12 padding-zero">
+          <div class="">
+            <div class="img panel-img">
+              <%= display_picture g %>
+            </div>
+
+            <% if (user_is_admin? || user_signed_in? && g.user==current_user) %>
+              <div class= "overlay-btn">
+              <%= link_to (fa_icon "edit"), edit_game_path(g), :class=>"btn btn-primary" %>
+              <%= link_to (fa_icon "gamepad"), new_game_key_path(g), :class=>"btn btn-primary" %>
+              <%= link_to (fa_icon "trash"), game_path(g),
+                                   method: :delete,
+                                   data: {confirm: 'Are you sure?'}, :class=>"btn btn-primary" %>
+              </div>
+            <% end %>
+
+          </div>
+          <div class="tag-boxes">
+            <% g.tags.each do |t| %>
+              <span><%= t.name %></span>
+            <% end %>
+          </div>
+          <div class="col-md-12" id="padding-zero">
```

Comment:
I'm pretty sure that this `div` is unnecessary. Always try to reduce complexity when possible